app/models/concerns/account/suspensions.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Account::Suspensions
+  extend ActiveSupport::Concern
+
+  included do
+    scope :suspended, -> { where.not(suspended_at: nil) }
+    scope :without_suspended, -> { where(suspended_at: nil) }
+  end
+
+  def suspended?
+    suspended_at.present? && !instance_actor?
+  end
+  alias unavailable? suspended?
+
+  def suspended_permanently?
+    suspended? && deletion_request.nil?
+  end
+
+  def suspended_temporarily?
+    suspended? && deletion_request.present?
+  end
+  alias permanently_unavailable? suspended_permanently?
```

Comment:
This should probably be up on line 18/19? (if the alias's are meant to be with the method declaration they alias?)

Reply:
Probably should, I just copied it as-is. Updated to move up.